qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageAppointmentsPage.java

```diff
@@ -14,38 +14,38 @@
 
 public class ManageAppointmentsPage extends Page {
 
-	public static final By DELETE_REQUEST = By.cssSelector(".delete-item.icon-remove");
-	private static final By CANCEL_BUTTON = By.cssSelector("#searchButtons > button");
-	private static final By SEARCH = By.cssSelector("#searchButtons > button.confirm");
+	public static final By DELETE_REQUEST_ICON = By.cssSelector(".delete-item.icon-remove");
 	private static final By APPOINTMENT = By.xpath("//table[@id='appointmentTable']/div[2]/div/div/div/div[2]/div[2]/div");
-	private static final By NEXT = By.cssSelector("#selectAppointment > button.confirm");
-	private static final By SAVE = By.xpath("//div[@id='confirmAppointment']/div[2]/button[2]");
-	private static final By BOOK_APPOINTMENT = By.cssSelector("i.icon-calendar:nth-child(1)");
-	private static final By YES_DELETE_REQUEST = By.xpath("//div[@id='confirm-cancel-appointment-request']/div[2]/button");
-	private static final By VIEW_ALL_TYPES_LINK = By.cssSelector("#viewAllAppointmentTypes > a");
+	private static final By SAVE_BUTTON = By.xpath("//div[@id='confirmAppointment']/div[2]/button[2]");
+	private static final By CANCEL_BUTTON = By.cssSelector("#searchButtons button");
+	private static final By SEARCH_BUTTON = By.cssSelector("#searchButtons button.confirm");
+	private static final By NEXT_BUTTON = By.cssSelector("#selectAppointment button.confirm");
+	private static final By BOOK_APPOINTMENT_ICON = By.cssSelector("i.icon-calendar:nth-child(1)");
+	private static final By CONFIRM_CANCEL_APPOINTMENT_REQUEST_BUTTON  = By.xpath("//div[@id='confirm-cancel-appointment-request']/div[2]/button");
```

Comment:
and here

Reply:
Sure

Reply:
@kdaud  I have attended to the reviews and pushed a commit,  please you could make more.